### ISSUE TITLE
Edit Custom filters shows all filters in Local folders (should only show filter templates)

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#204]
When opening Custom Templates / Edit from the advanced tab, we expect to see a list of quickFilters custom templates only

![image](https://github.com/RealRaven2000/quickFilters/assets/6314674/21cef986-2759-4333-a36f-394aeab11c2f)

instead _all_ filters in the Local Folders are shown. In error console, we see the message:

```log
Uncaught TypeError: fWin.quickFilters is undefined
    filterCustomTemplates chrome://quickfilters/content/qFilters-utils.js:1759
    filterCustomTemplates chrome://quickfilters/content/qFilters-utils.js:1779
qFilters-utils.js:1759:18
    filterCustomTemplates chrome://quickfilters/content/qFilters-utils.js:1759
    filterCustomTemplates chrome://quickfilters/content/qFilters-utils.js:1779

```

**Comment by RealRaven2000** [comment]
This is possibly caused by the filter window not having been fully loaded (quickFilters injects itself via a chrom overlay), so it might be easily fixed by adding a delay of 250ms.

**Comment by RealRaven2000** [comment]
Fixed in 5.9.4 - Published 13/08/2023
also in 6.2 - Published 13/08/2023
